inventree_digikey/Inventree.py

```diff
@@ -86,27 +122,32 @@ def create_inventree_part(dkpart: DigiPart):
     part = Part.create(API, {
         'name': dkpart.name,
         'description': dkpart.description,
-        'category': category.pk,
+        'category': category,
         'active': True,
         'virtual': False,
         'component': True,
         'purchaseable': 1
         })
     upload_picture(dkpart, part)
+    set_parameters(dkpart, part)
     return part
 
 
-def find_category():
+def find_category(dkpart: DigiPart):
     if API is None:
         load_config()
-    categories = PartCategory.list(API)
-    print("="*20)
-    print("Choose a category")
-    for idx, category in enumerate(categories):
-        print("\t%d %s" %(idx, category.name))
-    print("="*20)
-    idx = int(input("> "))
-    return categories[idx]
+    if str.lower(dkpart.category) in catagory_map:
+        return catagory_map[str.lower(dkpart.category)]
+    else:
```

Comment:
```suggestion
    clean_category = str.lower(dkpart.category)
    if clean_category in catagory_map:
        return catagory_map[clean_category]
    else:
```
This is a nitpick, not required if you don't want to

Reply:
Good catch.